src/gui/src/mainWindow.cpp

```diff
@@ -191,7 +191,7 @@ void MainWindow::createActions()
   zoom_out_ = new QAction("Zoom out", this);
   zoom_out_->setShortcut(QString("Shift+Z"));
 
-  congestion_setup_ = new QAction("Congestion Setup...");
+  congestion_setup_ = new QAction("Congestion Setup...", this);
```

Comment:
What version of Qt are you using?  This should match:

QAction::QAction(const QString &text, QObject *parent = nullptr)
See https://doc.qt.io/qt-5/qaction.html#QAction-1

Reply:
Normally the parent is used for a QActionGroup.

Reply:
Hi Matt, thanks for the review. 

My qt is v5.5.1, and I corrrected the fix from "this" to nullptr.

```
$> qmake -v
QMake version 3.0
Using Qt version 5.5.1 in /usr/lib/x86_64-linux-gnu
```